Create project form

d-ang1993 opened the pull request:
Functional

d-ang1993 committed:
added textfields with onChange event handler

d-ang1993 committed:
added textfields with onChange event handler

d-ang1993 committed:
added delete functionality to reactDropZone

d-ang1993 committed:
added delete functionality to reactDropZone

wamts reviewed (approved):
Submit ('subscribe') should call onSubmit

wamts committed:
Made uploadProj to hit project route

CreateProjectFormHook now converts files to base64 to send to backend

Increased upload limit using 'bodyParser'

Disabled some functionality in POST projects until we get S3 working

wamts committed:
Merge branch 'createProjectForm' of https://github.com/hatchways/team-macchiato into createProjectForm

d-ang1993 committed:
Merge branch 'dev' into createProjectForm
